plugins/openweather/README.md

```diff
@@ -0,0 +1,52 @@
+# API openweather client for AresMush
+
+This is a plugin for real weather reporting for game areas.
+It's based off [OpenWeatherLite](https://github.com/zsyed91/) Ruby library.
+
+# Requirements
+* [An OpenWeather Map API Key](https://openweathermap.org/)
+
+# Configuration
+1. Put this plugin in ~/aresmush/plugins/openweather
```

Comment:
Running `plugin/install openweather` will do steps 1&2 for them :)

Reply:
Nice, updated the documentation.